packages/fxa-payments-server/src/components/LoadingOverlay.tsx

```diff
@@ -19,8 +15,3 @@ export const LoadingOverlay = ({ isLoading }: LoadingOverlayProps) =>
       <img alt="Loading" src={spinnerImage} />
     </div>
   );
-
-/* istanbul ignore next - skip testing redux connect() */
-export default connect((state: State) => ({
-  isLoading: selectors.isLoading(state),
-}))(LoadingOverlay);
```

Comment:
I removed this from our selectors, as it wasn't being hit:

```
isLoading = state => Object.values(state.api).some(v => v && !!v.loading);
```

It was showing as uncovered in tests, as we only ever directly pass in `isLoading={true}` or `isLoading={false}` such as the following in `Subscriptions/index.tsx`:

```
  if (
    customer.loading ||
    subscriptions.loading ||
    profile.loading ||
    plans.loading
  ) {
    return <LoadingOverlay isLoading={true} />;
  }
```

With our checks as they are, I don't think this needs to be connected to the store.

Reply:
:+1: I think this might be a leftover from when we displayed a general loading indicator without checking the individual API loading states